src/lib/tutorials/normalize-product-like-slug.ts

```diff
@@ -21,7 +21,7 @@ export function normalizeSlugForDevDot(productSlug: string): ProductSlug {
 	} else if (isProductSlug(productSlug)) {
 		return productSlug
 	} else {
-		throw new Error(
+		console.warn(
```

Comment:
This error is currently bringing down our tutorial-map endpoint (which we use for all of our link rewrites) since this tutorial map processes all tutorials with product slugs that aren't in beta.